sendgrid/helpers/mail/mail.py

```diff
@@ -251,6 +251,36 @@ def add_custom_arg(self, custom_arg):
             self._custom_args = []
         self._custom_args.append(custom_arg)
 
+################################################################
+
+
+class SendGridMessage(Mail):
+    def __init__(self, 
+                from_email=None,
+                subject=None, 
+                to_email=None, 
+                plain_text_content=None, 
+                html_content=None):
+        self.from_email = from_email
+        self.subject = subject
+        self.to_email = to_email
+        self.plain_text_content = plain_text_content
+        self.html_content = html_content
+
+    def get(self):
+        if self.plain_text_content is None:
+            self.plain_text_content = ""
+        if self.html_content is not None:
+            self.content = Content("text/html", self.plain_text_content + self.html_content)
+        else:
+            self.content = Content("text/plain", self.plain_text_content)
```

Comment:
About this. I tried adding the plain text content and html content separately like this:-
```
mail = Mail()
mail.add_content(Content("text/plain", "some text here"))
mail.add_content(
            Content(
                "text/html",
                "<html><body>some text here</body></html>"))
....
```
I sent the mail successfully through SendGrid, but the plain text part was not rendered in the email. Though I saw the original content of the mail and found the plain text part to be there. (Maybe a problem with gmail? ). So, to make the plain text part appear , I added both the parts and then both of them are being rendered.